packages/js/src/plugins/auctionHouseModule/AuctionHouseClient.ts

```diff
@@ -46,6 +56,56 @@ export class AuctionHouseClient {
     protected readonly auctioneerAuthority?: Signer
   ) {}
 
+  cancelBid(
+    input: WithoutAH<CancelBidInput>
+  ): Task<CancelBidOutput & { bid: Option<Bid> }> {
+    return new Task(async (scope) => {
+      const output = await this.metaplex
+        .operations()
+        .execute(cancelBidOperation(this.addAH(input)));
+      scope.throwIfCanceled();
+
+      if (input.bid.receiptAddress) {
+        try {
+          const bid = await this.findBidByAddress(
+            input.bid.tradeStateAddress
+          ).run(scope);
+
+          return { bid, ...output };
+        } catch (error) {
+          // Fallback to manually creating a listing from inputs and outputs.
+        }
+      }
+
+      return { ...output, bid: null };
+    });
+  }
```

Comment:
Let's not fetch the model after cancelling it here.

I've recently added a design principle on the SDK that **only fetches an account after writing to the cluster if it was created**. Otherwise, we just send the write operation and expect the user to explicitly refresh the model if they need to use the read operations.

So here this should be enough.

```suggestion
  cancelBid(
    input: WithoutAH<CancelBidInput>
  ): Task<CancelBidOutput> {
    return this.metaplex
      .operations()
      .getTask(cancelBidOperation(this.addAH(input)));
  }
```

Same goes for `cancelListing`.